packages/api/tests/marketDirectoryInfo.test.ts

```diff
@@ -67,6 +72,28 @@ const marketDirectoryInfo = mocks.fakeMarketDirectoryInfo({
   x: faker.internet.userName(),
 });
 
+type CompleteMarketDirectoryInfoReq = WithoutFarmId<WithoutId<MarketDirectoryInfo>> & {
```

Comment:
The difference between this and the imported `MarketDirectoryInfoReqBody` is that the imported sets the all properties to optional and this one expects a valid req body (one that should succeed).